src/main/resources/manifest.json

```diff
@@ -0,0 +1 @@
+{"guid":"FUspeHguZEO1acdrBXhMjQ","version":"1.0","permissions":[]}
```

Comment:
I guess the guid maybe sensitive for deployment i guess please leave it to local only.